ambari-web/app/app.js

```diff
@@ -61,103 +61,21 @@ module.exports = Em.Application.create({
    */
   isKerberosEnabled: false,
 
-  /**
-   * state of stack upgrade process
-   * states:
-   *  - NOT_REQUIRED
-   *  - PENDING
-   *  - IN_PROGRESS
-   *  - HOLDING
-   *  - COMPLETED
-   *  - ABORTED
-   *  - HOLDING_FAILED
-   *  - HOLDING_TIMEDOUT
-   * @type {String}
-   */
-  upgradeState: 'NOT_REQUIRED',
```

Comment:
These enums are still valid for upgrades - maybe you should extract them out into a constant somewhere?

Reply:
Good idea. Done.